tensorflow/contrib/receptive_field/python/util/receptive_field.py

```diff
@@ -52,10 +52,22 @@ def _stride_size(node):
     stride_x: Stride size for horizontal direction (integer).
     stride_y: Stride size for vertical direction (integer).
   """
-  strides_attr = node.attr["strides"]
-  logging.vlog(4, "strides_attr = %s", strides_attr)
-  stride_y = strides_attr.list.i[1]
-  stride_x = strides_attr.list.i[2]
+  if node.op == "MaxPoolV2":
+    strides_input_name = node.input[2]
+    if not strides_input_name.endswith("/strides"):
+      raise ValueError("Strides name does not end with '/strides'")
+    strides_node = name_to_order_node[strides_input_name].node
+    if strides_node.op != "Const":
```

Comment:
Why does it need to be Const? Isn't the point of the new version that it can take a computed tensor value as argument?

Reply:
@rmlarsen Removed. Thanks.